lib/matplotlib/pyplot.py

```diff
@@ -2690,7 +2690,13 @@ def matshow(A: ArrayLike, fignum: None | int = None, **kwargs) -> AxesImage:
     return im
 
 
-def polar(*args, **kwargs) -> list[Line2D]:
+def polar(
+        *args: float | ArrayLike | str,
+        scalex: bool = True,
```

Comment:
I would also leave `scalex`, `scaley` out. They are an odd wart of `plot`. No other plotting method has this and I'd rather not explicitly talk about it in `polar`, even if this means it's not type annotated.